mongodb/create_poet_db.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Will this script be only used in tests, or also in deployments for production?

Reply:
I'm guessing production would be different - I'm not 100% clear on how production is set up (are we using Atlas?).

Reply:
Yes. We are using Atlas.

Ping @WesleyCharlesBlake regarding the use of the script (in production and QA).

Reply:
this script will not be used on the Atlas mongodb clusters. Atlas allows us to setup users, and specify perms on DBs and collections through its UI, and the DB's get created when we specify the connection URI in the mongo client (ie `mongodb+srv://admin:<PASSWORD>@<some-cluster-id>.mongodb.net/test` will end up creating a `test` db, provided that the credentials provided in the URI have permissions to read/write on that DB.

from a K8s perspectice, we supply the mongo Atlas DB creds in values.yaml, and this creates a config secret in K8s which the node deployment then retrieves to configure mongo auth

Reply:
OK. Thanks @geoffturk and @WesleyCharlesBlake.

We should move this file to /tests then. /scripts could work too, if we intend to run it manually.

Reply:
I disagree - this is not for tests, it is for bootstrapping a brand new instance of the mongo docker container. It's a hook into how the mongo containers work so you can have an existing set of databases and users already set up when the container starts.